src/java/org/apache/cassandra/schema/CompressionParams.java

```diff
@@ -314,7 +314,7 @@ private static Class<?> parseCompressorClass(String className) throws Configurat
         if (className == null || className.isEmpty())
```

Comment:
This class is missing a crucial change: min compressed ratio / max compressed length cannot apply for encrypted files.

See https://github.com/riptano/bdp/blob/a4c8076bc0b0d293958e94189cb0ec7e5c796319/dse-db/src/java/org/apache/cassandra/schema/CompressionParams.java#L554-L564

Perhaps we should also add a check somewhere that these limits are not in play (e.g. in `CompressedChunkReader`) if a compressor includes an encryptor.